native/src/components/PrimaryTextButton.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { ReactElement } from 'react'
+import { Button } from 'react-native-elements'
+import { IconNode } from 'react-native-elements/dist/icons/Icon'
+import styled, { useTheme } from 'styled-components/native'
+
+const Container = styled.View<{ $padding: boolean }>`
+  padding: ${props => (props.$padding ? '16px' : '0')};
+`
+
+type PrimaryTextButtonProps = {
+  onPress: () => void
+  text: string
+  disabled?: boolean
+  Icon?: IconNode
+  padding?: boolean
+}
+
+const PrimaryTextButton = ({
```

Comment:
I think its better to call it TextButton and pass a `variant` that can be primary, secondary, whatever.
I think you should talk with Toni about the design system. Now we refactor and later we refactor again when different text button variants comes up. I think this should take place in a `component/base` folder